cou/cli.py

```diff
@@ -82,25 +82,27 @@ def setup_logging(log_level: str = "INFO") -> None:
     :rtype: None
     """
     log_formatter = logging.Formatter(
-        fmt="%(asctime)s [%(levelname)s] %(message)s", datefmt="%Y-%m-%d %H:%M:%S"
+        fmt="%(asctime)s [%(name)s] [%(levelname)s] %(message)s", datefmt="%Y-%m-%d %H:%M:%S"
```

Comment:
I think that we should use use logger name in our logs, because it will be more simpler to see from where those logs are coming. However we are using filter for console handler, which will block all other logs, and I do not see how we will benefit from seeing logger name there.
We should use simple logging format for console handler and more complicated (with logger name or other attributes) for file handler.